doc/whats_new/v1.1.rst

```diff
@@ -537,6 +537,12 @@ Changelog
   `alpha`, `max_iter` and `tol`.
   :pr:`22240` by :user:`Arturo Amor <ArturoAmorQ>`.
 
+- |Enhancement| :class:`linear_model.GeneralizedLinearRegressor` fit method
```

Comment:
`GeneralizedLinearRegressor` is not part of the public API, `PoissonRegressor`, `GammaRegressor` and `TweedieRegressor` are public.